docs/api/touch-bar.md

```diff
@@ -2,7 +2,7 @@
 
 > Create TouchBar layouts for native macOS applications
 
-Process: [Main](../tutorial/application-architecture.md#main-and-renderer-processes)
```

Comment:
It seems that changing this line would cause typescript errors:

```
ERROR in C:\projects\src\electron\lib\browser\api\browser-window.ts
5109[tsl] ERROR in C:\projects\src\electron\lib\browser\api\browser-window.ts(1,55)
5110      TS2305: Module '"electron/main"' has no exported member 'TouchBar'.
5111
```

@MarshallOfSound Do you have an idea on how to fix this?